src/Infrastructure/MembershipApplicationTrackingInfo.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Infrastructure;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class MembershipApplicationTrackingInfo {
+
+	private $totalImpressionCount;
+	private $singleBannerImpressionCount;
+	private $confirmationPage;
+	private $confirmationPageCampaign;
```

Comment:
- `$singleBannerImpressionCount`: number of times the current banner was shown (before the user decided to donate)
- `$totalImpressionCount`: total number of times any banner was shown
- `$confirmationPageCampaign` and `$confirmationPage`: similar to the banner tracking campaign and keyword; confirmation pages may embed modified membership forms to a/b test against

The first two don't need to be tracked, as this is actually only relevant for the donation tracking. The latter two can/should be combined into one (`campaign/keyword`).